volue_insight_timeseries/__init__.py

```diff
@@ -6,6 +6,15 @@
 from .session import Session
 from . import auth, curves, events, session, util
 
+__all__ = [
```

Comment:
why do you need this? If it is for importing convenience, `volue_insight_timeseries` is a package already and IDE should be able to recognize those modules.

Reply:
I was thinking about this for a long time whether to include it or not, but in my opinion the benefit is that we explicitly say, what are the public facing parts of `vit`. Also in my opinion it improves readability as well.

Reply:
I am not aware of this practice. Do you have some document that I could refer to? This implicitly indicates new modules also need to be added here, which can be easily missed and cause confusion. Lets get more opinions on this internally.

Reply:
It is nice and clean way of telling users what to use, yes. But there could be customers who imported other stuff, and then their code would break if they upgrade to this version. So introducing this, could mean we would need to release this as a new major version. Could you extract it to a separate PR?

Reply:
And as Stig mentioned I will make it a different PR altogether.